server/routes/incentiveMigration/incentivesMigrationController.ts

```diff
@@ -34,6 +35,8 @@ export default class IncentivesMigrationController {
     } else {
       const { migrations } = await this.nomisMigrationService.getIncentivesMigrations(context(res), {
         ...searchFilter,
+        toDateTime: withDefaultTime(searchFilter.toDateTime),
+        fromDateTime: withDefaultTime(searchFilter.fromDateTime),
       })
```

Comment:
without this users are forced to add time element that is a pain, was already present for visits but missed it from incentives